modular/blocksyncer/modules/registrar.go

```diff
@@ -0,0 +1,51 @@
+package registrar
```

Comment:
blocksyncer/modular/modules seems not good name.
modules is special for ‘metadata’, 'uploader' etc. 
here modules have some confusion with the above.

Reply:
"We have expanded the interface of Juno in SP and implemented it. In Juno, every data module is conceptualized as 'modules', so we have aligned it here.
cc @krish-nr

Reply:
<img width="1474" alt="image" src="https://github.com/bnb-chain/greenfield-storage-provider/assets/122767193/b1c9e1b4-9fab-4e41-9b1c-474471c8cb86">